odd-platform-ui/src/redux/actions/labels.actions.ts

```diff
@@ -1,27 +1,23 @@
-import { createAsyncAction } from 'typesafe-actions';
-import { LabelsResponse, Label } from 'generated-sources';
-import { PaginatedResponse } from 'redux/interfaces/common';
+import { createActionType } from 'lib/redux/helpers';
 
-export const fetchLabelsAction = createAsyncAction(
-  'GET_LABELS__REQUEST',
-  'GET_LABELS__SUCCESS',
-  'GET_LABELS__FAILURE'
-)<undefined, PaginatedResponse<LabelsResponse>, undefined>();
+export const labelsActionPrefix = 'labels';
 
-export const createLabelsAction = createAsyncAction(
-  'POST_LABELS__REQUEST',
-  'POST_LABELS__SUCCESS',
-  'POST_LABELS__FAILURE'
-)<undefined, Label[], undefined>();
+export const fetchLabelsAction = createActionType(
+  labelsActionPrefix,
+  'fetchLabels'
+);
```

Comment:
Somewhere in the project you can find constructions like this and it has an inaccuracy in the naming. Here we exactly creating an action types, not actions.

```suggestion
export const fetchLabelsActionType = createActionType(
  labelsActionPrefix,
  'fetchLabels'
);
```